Makefile

```diff
@@ -247,7 +247,7 @@ build-attestor-docker: ## Build scorecard-attestor Docker image
 build-attestor-docker:
 	DOCKER_BUILDKIT=1 docker build . --file attestor/Dockerfile \
 		--tag scorecard-attestor:latest \
-		--tag scorecard-atttestor:$(GIT_HASH)
+		--tag scorecard-attestor:$(GIT_HASH)
```

Comment:
I'm uncertain about this... It seems quite strange...

Reply:
Should be fine, thanks for the fix!